### PR TITLE
add landing message. #1

### PR DESCRIPTION
**Pull request by tatdef**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,6 @@
   <title>Welcome to Potionfy</title>
 </head>
 <body>
-   
+   <H1>Welcome to Potionfy ! Enjoy your stay.<H1>
```
[the remark this answers is not in the document]
Comment (tatdef) [reply index.html:10]: ok ill do it

**Review by tatdef — COMMENTED** [review]
helo

**Review by ho757uff — CHANGES_REQUESTED** [review]
Please change !

**Review by ho757uff — APPROVED** [review]
Approved !

**Pull request merged**